test/translator/inquiry_test.cc

```diff
@@ -77,23 +77,25 @@ TEST_F(InquiryTest, InquiryToNvme) {
   translator::Allocation allocations[2] = {{}};
 
   translator::StatusCode status = translator::InquiryToNvme(
-      scsi_cmd_, nvme_cmds_[1], nvme_cmds_[0], alloc_len, nsid, allocations);
+      scsi_cmd_, nvme_cmds_[0], nvme_cmds_[1], alloc_len, nsid, allocations);
```

Comment:
Change in ordering is deliberate, see PR description https://github.com/googleinterns/scsi2nvme/pull/164#issue-460531001